util/http/http.go

```diff
@@ -2,27 +2,120 @@ package http
 
 import (
 	"fmt"
+	"math"
 	"net/http"
 	"net/http/httputil"
+	"strconv"
 	"strings"
 
 	log "github.com/sirupsen/logrus"
 )
 
+// max number of chunks a cookie can be broken into. To be compatible with
+// widest range of browsers, we shouldn't create more than 30 cookies per domain
+const maxNumber = 5
```

Comment:
Package name is `http` so it is not just about cookies. Please rename variables to denote this is related to cookies. E.g. `maxCookieNumber`, `maxCookieLength`